### DISCUSSION TITLE
Dificuldade em iniciar o repositório

### DISCUSSION THREAD

**Discussion by vagner-begni** [campuscode/rails-guides-pt-BR#830] [Geral]
Boa noite!
Estou iniciando minha tentativa de contribuir com este projeto, que conheci durante o QSD40+ na CampusCode. Mas estou tendo dificuldade logo de início, na hora de clonar o repositório. Conforme o guia inicial, digito o comando:

git clone git@github.com:vagnersbegni/rails-guides-pt-BR.git        , onde vagnersbegni é meu username no GitHub.

E aparece a seguinte mensagem:
Cloning into 'rails-guides-pt-BR'...
ERROR: Repository not found.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

Agradecendo já a atenção, fico à disposição para qualquer informação adicional necessária.

**Comment by HenriqueMorato** [comment]
Oi, @vagnersbegni bem vindo ao guia. Não vi na sua conta o repo como publico (pode ser que esteja privado) mas confere se fez o `Fork` clicando no botão lá em cima a direita, assim você vai fazer a copia pra sua conta e aí você vai conseguir fazer o `clone`

**Comment by vagner-begni** [comment]
Obrigado, Henrique!
Eu tinha feito o fork há um tempo em uma conta no github. E agora estava fazendo o clone em outra. Pronto, tudo numa conta só e clone feito...
Muito obrigado e foi mau pela confusão.
